[2024-05-14T07:47:08.787+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-14T07:47:08.803+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline_jobs104.data_crawler_list manual__2024-05-14T07:47:07.753695+00:00 [queued]>
[2024-05-14T07:47:08.809+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline_jobs104.data_crawler_list manual__2024-05-14T07:47:07.753695+00:00 [queued]>
[2024-05-14T07:47:08.809+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-14T07:47:08.818+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): data_crawler_list> on 2024-05-14 07:47:07.753695+00:00
[2024-05-14T07:47:08.822+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=574) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-14T07:47:08.823+0000] {standard_task_runner.py:63} INFO - Started process 576 to run task
[2024-05-14T07:47:08.823+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline_jobs104', 'data_crawler_list', 'manual__2024-05-14T07:47:07.753695+00:00', '--job-id', '53', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline_jobs104.py', '--cfg-path', '/tmp/tmpeenymstg']
[2024-05-14T07:47:08.824+0000] {standard_task_runner.py:91} INFO - Job 53: Subtask data_crawler_list
[2024-05-14T07:47:08.847+0000] {task_command.py:426} INFO - Running <TaskInstance: data_pipeline_jobs104.data_crawler_list manual__2024-05-14T07:47:07.753695+00:00 [running]> on host 9bf348f884a5
[2024-05-14T07:47:08.889+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline_jobs104' AIRFLOW_CTX_TASK_ID='data_crawler_list' AIRFLOW_CTX_EXECUTION_DATE='2024-05-14T07:47:07.753695+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-14T07:47:07.753695+00:00'
[2024-05-14T07:47:08.890+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-14T07:47:08.901+0000] {logging_mixin.py:188} INFO - 設定排除44家公司
[2024-05-14T07:47:10.252+0000] {logging_mixin.py:188} INFO - url: https://www.104.com.tw/jobs/search/?ro=1&keyword=%E5%BE%8C%E7%AB%AF%E5%B7%A5%E7%A8%8B%E5%B8%AB+python&isnew=3&jobexp=1%2C3&mode=l&order=16&asc=0
[2024-05-14T07:47:10.256+0000] {logging_mixin.py:188} INFO - 執行錯誤, retry 1, Message: Unable to obtain driver for chrome; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location
[2024-05-14T07:47:11.334+0000] {logging_mixin.py:188} INFO - url: https://www.104.com.tw/jobs/search/?ro=1&keyword=%E5%BE%8C%E7%AB%AF%E5%B7%A5%E7%A8%8B%E5%B8%AB+python&isnew=3&jobexp=1%2C3&mode=l&order=16&asc=0
[2024-05-14T07:47:11.336+0000] {logging_mixin.py:188} INFO - 執行錯誤, retry 2, Message: Unable to obtain driver for chrome; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location
[2024-05-14T07:47:12.402+0000] {logging_mixin.py:188} INFO - url: https://www.104.com.tw/jobs/search/?ro=1&keyword=%E5%BE%8C%E7%AB%AF%E5%B7%A5%E7%A8%8B%E5%B8%AB+python&isnew=3&jobexp=1%2C3&mode=l&order=16&asc=0
[2024-05-14T07:47:12.405+0000] {logging_mixin.py:188} INFO - 執行錯誤, retry 3, Message: Unable to obtain driver for chrome; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location
[2024-05-14T07:47:12.406+0000] {logging_mixin.py:188} INFO - 達到重試上限，放棄爬取
[2024-05-14T07:47:12.414+0000] {logging_mixin.py:188} INFO - 過濾剩0筆資料 | 花費 3.51 秒
[2024-05-14T07:47:12.415+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-14T07:47:12.417+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-14T07:47:12.436+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_pipeline_jobs104, task_id=data_crawler_list, run_id=manual__2024-05-14T07:47:07.753695+00:00, execution_date=20240514T074707, start_date=20240514T074708, end_date=20240514T074712
[2024-05-14T07:47:12.469+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-14T07:47:12.487+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-14T07:47:12.488+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
